functional-tests/src/test/groovy/FrontendCore/modules/Tab_Permit.groovy

```diff
@@ -3,13 +3,39 @@ package modules
 import geb.Module 
 
 class Tab_Permit extends Module {
+    static PERMIT_NUMBER = "M-66666"
     static at = {activeTab=='Permit'}
     static content = {
         activeTab (wait:true) {$("div.ant-tabs-tab-active").text()}
         tabSelect (wait:true) {$("div.ant-tabs-tab", text: "Permit")}
 
         permit_no (wait:true) {$("td", text: contains("BLAHPER-01"), 0)}
+
+        //Permit Tab objects
+        newPermitButton (wait:true) {$("button", text: "Add a New Permit")}
+        newPermitForm { module Form_CreatePermit }
+        permitRow (wait:true) {$("tr").has("div", title: "Permit No.",text: PERMIT_NUMBER)}
+        permitTitle (wait:true) {$("div", title: "Permit No.",text: PERMIT_NUMBER)}
+        addEditButton (wait:true) {$("button").has("div", text: "Add/Edit")}
+        
+        hoverDropdown (wait:true) {$("ul", role:"menu").has("button", text: "Edit permit status").findAll { it.displayed } }
```

Comment:
These can probably all change from _findAll_ to _find_ - _find_ just returns the first result found without continuing the iteration, and it looks like you're ever expecting a single result to be found